src/components/application_manager/src/commands/mobile/system_request.cc

```diff
@@ -43,9 +45,356 @@ Copyright (c) 2013, Ford Motor Company
 #include "utils/file_system.h"
 #include "formatters/CFormatterJsonBase.hpp"
 #include "json/json.h"
+#include "utils/helpers.h"
 
 namespace application_manager {
 
+namespace {
+
+CREATE_LOGGERPTR_LOCAL(logger_, "ApplicationManager")
+
+using namespace application_manager::strings;
```

Comment:
@okoreniak  I think it is very bad idea to add this using. There to many redundant names. Also agccording GStyle using is not allowed